### ISSUE TITLE
using submodules

### ISSUE THREAD

**Issue by axd1967** [codepath/android_guides#82]
Is there a need to use submodules? The submodule concept is tricky, and this repo uses one submodule only. The contribution guide silently assumes a contributor masters the submodule concept.

**Comment by nesquena** [comment]
You are right, the submodule approach is not ideal. Need to come up with an easier way to allow people to send pull requests for the wiki.

**Comment by axd1967** [comment]
Why not move the current pages right into this repository, and ditch the submodule?

I guess the issue is clouded by the fact that the submodule is actually the GitHub wiki content (or that's probably the reason why a submodule is being used).

A possible, but less attractive alternative could be to ditch the wiki, and use the web publishing feature ("pages) of GitHub instead. (I'm not experienced in that for the moment, so I'm not sure how well that would work)

**Comment by yvolk** [comment]
+1 for moving content to a single repository. 
I just wanted to contribute my corrections to the guide directly from my device, but I can't - I don't have a User friendly tool for this on Android :-(

**Comment by axd1967** [comment]
hello??

this is a confusing situation.

here is an example of a problem: https://github.com/codepath/android_guides/wiki/Listening-to-Sensors-using-SensorManager is not the same as 04b67ba9e86616a4beeb2b6d286168e9e8eb72e6 (use this SHA-1 in the "other" wiki, and if this confuses you, you found the problem that this issues tries to uncover!)

![ag](https://cloud.githubusercontent.com/assets/3529789/21979017/713f17aa-dbdd-11e6-80ef-d7c548751967.png)
